### DISCUSSION TITLE
Can I choose to disable the display of "Popular Repositories" when I view my organization?

### DISCUSSION THREAD

**Discussion by gordonj7** [community/community#113496] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I go to `github.com/myorganization` in a browser, the first windowful of content is the "Popular Repositories" area.

These repositories are _literally never_ ones that I want to look at, and I always have to scroll past them.

Is there a way to configure my personal settings so that it does not show that content, and skips right to the regular "Repositories" section?

**Comment by nickkipshidze** [comment]
I am pretty sure you can pin some repositories. Have you tried pinning the repositories that you want to see on the front page?

**Reply by gordonj7** [reply]
I will try that.  Thanks!  I also noticed that selecting the "Repositories" tab at the top, instead of the "Overview" tab, is a persistent setting.
